xcp_d/tests/test_workflow_anatomical.py

```diff
@@ -0,0 +1,272 @@
+        "native_lh_midthickness": os.path.join(
+            tmpdir, "sub-01_hemi-L_desc-hcp_midthickness.surf.gii"
+        ),
+        "native_lh_vinflated": os.path.join(tmpdir, "sub-01_hemi-L_desc-hcp_vinflated.surf.gii"),
+        "native_lh_sulcal_depth": os.path.join(tmpdir, "sub-01_hemi-L_sulc.shape.gii"),
+        "native_lh_sulcal_curv": os.path.join(tmpdir, "sub-01_hemi-L_curv.shape.gii"),
+        "native_lh_cortical_thickness": os.path.join(tmpdir, "sub-01_hemi-L_thickness.shape.gii"),
+        "native_rh_inflated": os.path.join(tmpdir, "sub-01_hemi-R_desc-hcp_inflated.surf.gii"),
+        "native_rh_midthickness": os.path.join(
+            tmpdir, "sub-01_hemi-R_desc-hcp_midthickness.surf.gii"
+        ),
+        "native_rh_vinflated": os.path.join(tmpdir, "sub-01_hemi-R_desc-hcp_vinflated.surf.gii"),
+        "native_rh_sulcal_depth": os.path.join(tmpdir, "sub-01_hemi-R_sulc.shape.gii"),
+        "native_rh_sulcal_curv": os.path.join(tmpdir, "sub-01_hemi-R_curv.shape.gii"),
+        "native_rh_cortical_thickness": os.path.join(tmpdir, "sub-01_hemi-R_thickness.shape.gii"),
+    }
+
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_midthickness"])
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_inflated"])
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_vinflated"])
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_sulcal_depth"])
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_sulcal_curv"])
+    shutil.copyfile(files["native_lh_pial"], files["native_lh_cortical_thickness"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_midthickness"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_inflated"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_vinflated"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_sulcal_depth"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_sulcal_curv"])
+    shutil.copyfile(files["native_rh_pial"], files["native_rh_cortical_thickness"])
+
+    final_files = files.copy()
+    for fref, fpath in files.items():
+        std_fref = fref.replace("native_", "fsLR_")
+        std_fname = os.path.basename(fpath)
+        std_fname = std_fname.replace("sub-01_", "sub-01_space-fsLR_den-32k_")
+        std_fpath = os.path.join(tmpdir, std_fname)
+        shutil.copyfile(fpath, std_fpath)
+        final_files[std_fref] = std_fpath
+
+    return final_files
+
+
+def test_init_warp_surfaces_to_template_wf_01(
+    datasets,
+    fmriprep_with_freesurfer_data,
+    surface_files,
+    tmp_path_factory,
+):
+    tmpdir = tmp_path_factory.mktemp("test_init_warp_surfaces_to_template_wf_01")
+
+    subject_id = "01"
+    standard_spaces_available = {
+        "mesh": True,
+        "morphometry": True,
+        "shape": True,
+    }
+    surfaces_found = {
+        "mesh": True,
+        "morphometry": True,
+        "shape": True,
+    }
```

Comment:
@madisoth can you look through the combinations of `standard_spaces_available` and `surfaces_found` in these tests to see if they (1) are all possible and (2) cover all realistic combinations we'll get from either nibabies or fmriprep?